BrowserKit/Sources/ToolbarKit/AddressToolbar/AddressToolbar.swift

```diff
@@ -1,18 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/
-
-import Foundation
-
-/// Protocol representing an address toolbar.
-public protocol AddressToolbar {
```

Comment:
@thatswinnie can this be cleaned up? Same question for `protocol NavigationToolbar`

Reply:
@lmarceau this definitely has to stay. Same for the `NavigationToolbar` protocol.

Reply:
Reverted changes for this file